tests/test_pystockwatch.py

```diff
@@ -4,19 +4,19 @@
 def test_percent_change():
     # Invalid input for stock ticker
     with raises(NameError) as error_ticker:
```

Comment:
@helingogo we need to add those raises error message to your function rather than testing the inputs in test
Try to assert output of your test

Reply:
pystockwatch in front of the function everytime you call it is not needed, I have handled it in my code

Reply:
@sy25wang Raises error message will cause fail during the test. The error message already exist in the function, it should be fine to keep without error message here in the test.